kibbeh/src/lib/tests/constants.test.ts

```diff
@@ -0,0 +1,27 @@
+import { linkRegex } from "../constants";
+
+describe("Link Regex", () => {
+  test("Link Regex Test 1(Only Link)", () => {
```

Comment:
rename tests from `Link Regex Test 1(Only Link)` to just the part in the parens -> `Only Link`

Reply:
Done.